ui/src/app/pages/admin/user/admin-user-common.tsx

```diff
@@ -208,19 +211,11 @@ const displayModuleStatusAndDate = (
   profile: Profile,
   moduleName: AccessModule,
   child: string
-): JSX.Element => {
-  return (
-    <div style={moduleStatusStyle(getModuleStatus(profile, moduleName))}>
-      {child}
-    </div>
-  );
-};
-
-export const isBypassed = (
```

Comment:
this did a very similar thing to the other `isBypassed` so I removed it to avoid confusion.  That changed the signatures of the calls in this file.